plugin/src/main/kotlin/tanvd/grazi/EnglishChonker.kt

```diff
@@ -0,0 +1,44 @@
+package tanvd.grazi
+
+import com.intellij.openapi.application.ApplicationManager
+import org.jetbrains.org.objectweb.asm.*
+import org.jetbrains.org.objectweb.asm.Opcodes.ASM5
+
+
+class EnglishChonker(cv: ClassVisitor) : ClassVisitor(ASM5, cv) {
+    companion object {
+        fun reloadEnglish() {
+            val stream = GraziPlugin::class.java.getResourceAsStream("/org/languagetool/language/English.class")
+            val reader = ClassReader(stream)
+            val writer = ClassWriter(ClassWriter.COMPUTE_MAXS)
+            val visitor = EnglishChonker(writer)
+            reader.accept(visitor, 0)
+
+            val cls = writer.toByteArray()
+            with(ClassLoader::class.java.getDeclaredMethod("defineClass", String::class.java, ByteArray::class.java, Int::class.java, Int::class.java)) {
+                isAccessible = true
+
+                if (ApplicationManager.getApplication().isUnitTestMode) {
```

Comment:
It looks like we should handle this on side of `GraziPlugin.classLoader` :)

Reply:
Done